Homework 6/SortedList/SortedList/SortedList.h

```diff
@@ -0,0 +1,49 @@
+#pragma once
+#include <stdbool.h>
+
+typedef struct List List;
+
+typedef struct Position Position;
+
+// ������� ������� ������ - ���������� ��������� �� ��������� ListElement - ������ ������.
+List* createList();
+
+// ������� ������� ������. ��������� �� ���� ��������� �� ������ ������ (�� ��������� ListElement).
+// ������ �� ����������.
+void deleteList(List* list);
+
+// ������� ������� ������� � ������. ��������� �� ���� ��������� �� ������� ������ (��������� �� ��������� �� ��������� ListElement).
+// ������ �� ����������.
+void deletePosition(Position* position);
+
+// ������� ���������� ��������� �� ������ ������� ������. ��������� ��������� �� ������ ������.
+Position* first(List* list);
+
+// ������� ���������� ��������� �� ������� ������ �� ��������� ������� ������.
+// ��������� ��������� �� ������� ������� ������.
+void next(Position* position);
+
+// ������� ���������� true, ���� ������������ �������� - ��������� �� ������� ������ - �������� ���������� �� ��������� ������� ������.
+// false - ���� ������������ �������� - ��������� �� ������� ������ - �� �������� ���������� �� ��������� ������� ������.
```

Comment:
```suggestion
// false иначе.
```
Краткость --- не только сестра таланта, она ещё и уменьшает стоимость сопровождения, потому что каждый комментарий надо прочитать и понять.